api/controllers/main.js

```diff
@@ -32,15 +30,13 @@ module.exports = {
     }
     const context = defaultContext(req);
 
-    return Promise.all([
-      BuildCounter.countBuildsFromPastWeek(),
-      Site.count(),
-    ])
-      .then(([builds, sites]) => {
-        context.buildCount = builds;
-        context.siteCount = sites;
-        res.render('home.njk', context);
-      });
```

Comment:
This seemed left over from a prior era, don't see why we would want this on an unauthenticated page and don't think it is being used.